packages/common-scripts/src/deploy.ts

```diff
@@ -29,16 +29,20 @@ function updateOutputs(txSkeleton: TransactionSkeletonType, output: Cell): Trans
   return txSkeleton;
 }
 
-function updateCellDeps(txSkeleton: TransactionSkeletonType): TransactionSkeletonType {
+function updateCellDeps(txSkeleton: TransactionSkeletonType, config?: Config): TransactionSkeletonType {
   txSkeleton = txSkeleton.update('cellDeps', (cellDeps) => {
     return cellDeps.clear();
   });
-  const config = getConfig();
+  config = config || getConfig();
   const secp256k1Config = config.SCRIPTS.SECP256K1_BLAKE160!;
+  const secp256k1MultiSigConfig = config.SCRIPTS.SECP256K1_BLAKE160_MULTISIG!;
   txSkeleton = txSkeleton.update('cellDeps', (cellDeps) => {
     return cellDeps.push({
       out_point: { tx_hash: secp256k1Config.TX_HASH, index: secp256k1Config.INDEX },
       dep_type: secp256k1Config.DEP_TYPE,
+    }, {
+      out_point: { tx_hash: secp256k1MultiSigConfig.TX_HASH, index: secp256k1MultiSigConfig.INDEX },
+      dep_type: secp256k1MultiSigConfig.DEP_TYPE,
```

Comment:
```suggestion
      },
      // TODO optimize me, push dep directly without checking actual locks used would cause bigger tx
      {
        out_point: { tx_hash: secp256k1MultiSigConfig.TX_HASH, index: secp256k1MultiSigConfig.INDEX },
        dep_type: secp256k1MultiSigConfig.DEP_TYPE,
````

Reply:
need more specific description

Reply:
although it is allowed to add deps that are not referenced, this makes the tx larger

the CKB tx fee is calculated by the size of the tx, and the extra deps add to the size of the tx, even though the dep is just an OutPoint and not very large in itself

there may also be some security issues, although they are unlikely to occur.